woocommerce/woocommerce-ios
# [Auth] Remove the `.showLinkMailView` segue and navigate programmatically

This PR removes the segue that sends a user to the "Magic Link sent" screen and updates the programmatic navigation for opening the app after a Magic Link has been clicked.

See also: wordpress-mobile/WordPressAuthenticator-iOS#217

## To test:
1. Check out this branch
2. `rake dependencies`
3. Build and run
4. Navigate through the Login flow (listed below)

#### Login Flow
1. Select the Log in with Jetpack button
2. Enter a valid email address that can use Magic Links, select the Next button
3. Select the Send Link button 
4. Verify you are navigated to the "Magic link sent" screen
5. Verify you received an emailed magic link
6. Open the magic link
7. Verify the magic link navigates you to the "Continue" screen

### Do not merge until:
1. The Authenticator PR has been merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/217  ✔️
2. A new Authenticator release has been created  ✔️
3. The release is published on Cocoapods trunk ✔️
4. This PR points to the new release  ✔️


Update release notes:
